dynamo-store.js

```diff
@@ -594,8 +628,9 @@ function make_intern() {
 
             let sk = indexdefkey.sort
             if (null != sk && null != fq[sk]) {
-              listreq.KeyConditionExpression += ` and #${sk}n = :${sk}i`
-              listreq.ExpressionAttributeValues[`:${sk}i`] = fq[sk]
+              let fq_op = get_op(fq[sk])
```

Comment:
this could be null, throw if so

Reply:
The idea is to go with equality operator by default if this is `null`.

This won't break older functionalities. Thus, no `throw` 

see: https://github.com/senecajs/seneca-dynamo-store/blob/71f892a7491319ddfdedb975f3996ab7dd4b79ed/dynamo-store.js#L632

also see unit tests: https://github.com/senecajs/seneca-dynamo-store/blob/71f892a7491319ddfdedb975f3996ab7dd4b79ed/test/dynamo-store.test.js#L267

Reply:
but then you get equality if I use
```
foo: {notAValidOp$: 123}
```
this is an error
for this sort of code it is better to add additional condition checks since it is just naturally full of conditions, and we don't really want "defaults"

Reply:
fair enough! have another look, please.